.github/workflows/build_deploy.yml

```diff
@@ -214,7 +217,6 @@ jobs:
 
   build-and-publish-init-image:
     needs: [upload_pypi]
-    runs-on: ubuntu-latest
```

Comment:
Was this redundant?

Reply:
Not only redundant, it clashes with a top level `uses` which I suppose makes sense but isn't clear from their docs.